sycl/include/sycl/ext/oneapi/experimental/cuda/non_uniform_algorithms.hpp

```diff
@@ -10,11 +10,14 @@
 
 #if defined(__SYCL_DEVICE_ONLY__) && defined(__NVPTX__)
 #include "masked_shuffles.hpp"
+#include <sycl/ext/oneapi/experimental/non_uniform_groups.hpp>
 
 namespace sycl {
 inline namespace _V1 {
 namespace detail {
 
+using syclex = ::sycl::ext::oneapi::experimental;
```

Comment:
```suggestion
namespace syclex = sycl::ext::oneapi::experimental;
```
I suspect that will fix the build failures.

Reply:
failed CI jobs appear only for linux and about that experimental is not part of oneapi
I was just trying to change the path of aimed namespace.
And it fails only on CI...

i know i'ts not related - i'd replace it by working solution. Currently still I haven't done/reproduced such error locally